libibverbs/verbs.h

```diff
@@ -2078,6 +2127,8 @@ struct ibv_values_ex {
 
 struct verbs_context {
 	/*  "grows up" - new fields go here */
+	uint32_t log_level;
+	FILE *log_fp;
```

Comment:
Given that every context opens the same log_fp, this should not be in the struct verbs_context or in verbs.h..

Reply:
Where else? struct verbs_device?

Reply:
verbs_device is Ok, a global is good too.  Also keep in mind that opening the same file twice can be problematic, so better to have a global for the FILE *

Reply:
I'm thinking of changing both to global variables and initialize them in ibverbs_init(), though that means the file will only get closed implicitly on process cleanup, what do you think?